instagram/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PostsPage from './components/PostContainer/PostsPage';
+import Authenticate from './components/Authentication/Authenticate';
+
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: [],
```

Comment:
i don't believe posts on app is being used now that you have PostsPage or search.